### PR TITLE
feat:  human readable errors in state schema validation

### LINKED ISSUE #310: Human readable validation errors
# This is a Feature Proposal

## Description

* What is the use case that should be solved.

I want to read easily validation errors that comes from `asl-validator` so I can quickly fix them without having to decode JSON.

Here the output I get in case of error:

```
ServerlessError: ✕ State machine "machine1" definition is invalid:
  [{"keyword":"additionalProperties","dataPath":".States['Hello World']","schemaPath":"#/additionalProperties","params":{"additionalProperty":"Resource"},"message":"should NOT have additional properties"}, ...]
```

Here's an example of what it could like once formatted:
```
ServerlessError: ✕ State machine "machine1" definition is invalid:
  - ".States['Hello World']" should NOT have additional property "Resource" [additionalProperties]
  - ...
```

### PR DESCRIPTION
**Pull request by aslafy-z**
Leverage https://github.com/airware/asl-validator/pull/46 feature to show humanized version of errors to user (instead of raw JSON).

Depends on https://github.com/airware/asl-validator/pull/46
Fixes #310

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   },
   "dependencies": {
     "@hapi/joi": "^15.0.2",
-    "asl-validator": "^1.5.0",
+    "asl-validator": "github:aslafy-z/asl-validator#patch-1",
```
Comment (horike37) [inline package.json]: Why did you use your patch one? It would had better to be merged it to asl-validator. Is it possible?

Comment (aslafy-z) [reply package.json]: I'm waiting for my other PR to be merged and released before I can switch back the the real semver. Added a WIP tag in the title.

Comment (horike37) [reply package.json]: Thanks! Agreed with that 👍

**Comment by theburningmonk** [comment]
@aslafy-z looks like your change has been merged into the asl-validator as of v1.7.0, wanna come back and finish off this PR?

**Comment by aslafy-z** [comment]
Should be good by now! @theburningmonk 
Can you have a try from your side? I have no ready stack to do that in here. Thank you!

**Comment by theburningmonk** [comment]
@aslafy-z looks good, just tested it out on a stack

**Pull request merged**